parser/parser_test.go

```diff
@@ -584,6 +584,11 @@ func (s *testParserSuite) TestBuiltin(c *C) {
 		{"select curtime()", true},
 		{"select curtime(6)", true},
 
+		// select utc_timestamp
+		{"select utc_timestamp", false},
```

Comment:
This is valid in mysql:
mysql> select utc_timestamp;
+---------------------+
| utc_timestamp       |
+---------------------+
| 2017-02-05 02:10:07 |
+---------------------+
1 row in set (0.01 sec)